otcextensions/sdk/mrs/v1/cluster.py

```diff
@@ -0,0 +1,121 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from openstack import resource
+from openstack import _log
+
+from otcextensions.sdk import sdk_resource
+
+_logger = _log.setup_logging('openstack')
+
+
+class Cluster(sdk_resource.Resource):
+    resource_key = 'cluster'
+    resources_key = 'clusters'
+    base_path = '/cluster_infos'
+    # base_path = '/clusters/'
+    # list_path = '/cluster_infos'
+
+    # capabilities
+    allow_create = True
+    allow_list = True
+    allow_get = True
+    allow_delete = True
+    allow_update = True
+
+    _query_mapping = resource.QueryParameters(
+        'id', 'name', 'status',
+        'cluster_type', 'flavor', 'core_flavor', 'masternum',
+        'corenum', 'vpc', 'availability_zone', 'version', 'keypair')
+
+    #: Properties
+    # Specifies CLUSTER ID
```

Comment:
please describe props in a required format:
#: This is a new crazy parameter
#: *Type:str*